tests/documents/tags.org

```diff
@@ -7,7 +7,8 @@
 
 ** h12 : @Ti
 
-* h2 
+* h2  : tag2: h2:
 
-** h21 :#1:#2:
+** h21 :#1:#2: h2:
```

Comment:
Hey, could you please don't change the original tests, just add new case in a feature pr?